docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,219 @@
+
+    # Let's create a PermissionOverwrite instance that has some explicit permissions set:
+    overwrites = discord.PermissionOverwrite()
+    overwrites.update(send_messages=True, send_messages_in_threads=True)
+
+    # and now we update that with `manage_messages`:
+    overwrites.manage_messages = True
+
+    # we can now set those permissions on the channel:
+    await channel.set_permissions(moderator_role, overwrites=overwrites)
+
+This short example briefly goes over PermissionOverwrite objects, let's look more into those now:
+
+Using ``PermissionOverwrite`` objects during channel creation
+--------------------------------------------------------------
+
+We can also create channels with permissions set from the start.
+Let's create a text channel where only the moderator role, and someone trying to make a report can see the channel.
+
+We need to construct a mapping of ``Role | Member: PermissionOverwrite``:
+
+.. code-block:: python3
+
+    # We can now create the overwrites mapping necessary for this, you can do it in one dictionary instantiation, or spead it across multiple lines.
+
+    # I have added type annotations here so that the reader can understand what is happeining, and so type-checkers do too:
+    overwrites: Dict[Union[discord.Role, discord.Member], discord.PermissionOverwrite] = {
+        guild.default_role: discord.PermissionOverwrite(view_channel=False, send_messages=False, read_messages=False, read_message_history=False),
+        moderator_role: discord.PermissionOverwrite(manage_messages=True),
+        member: discord.PermissionOverwrite(view_channel=True, send_messages=True, read_messages=True, read_message_history=True)
+    }
+
+    # or alternatively:
+    overwrites: Dict[Union[discord.Role, discord.Member], discord.PermissionOverwrite] = {}
+    overwrites[guild.default_role] = discord.PermissionOverwrite(view_channel=False, send_messages=False, read_messages=False, read_message_history=False)
+    overwrites[moderator_role] = discord.PermissionOverwrite(manage_messages=True)
+    overwrites[member] = discord.PermissionOverwrite(view_channel=True, send_messages=True, read_messages=True, read_message_history=True)
+
+    # We can now pass these to channel creation:
+
+    await guild.create_text_channel(name="Ticket", overwrites=overwrites)
+
+The channel created whose permissions look something like this:
+
+.. image:: /images/guide/topics/permissions/ticket_channel.png
+
+Using ``PermissionOverwrite`` objects for finer control over channels
+----------------------------------------------------------------------
+
+:class:`~PermissionOverwrite` makes granting finer access to channels easier to manipulate.
+Let's create a scenario where you want to "lockdown" a channel, denying the ``@everyone`` role the permission to send messages.
+
+.. code-block:: python3
+
+    # I have found that the easiest way to do this, is to get the current overwrites for the Role or Member we're editing:
```

Comment:
Is it the easiest way? This is kind of a rhetorical question, but I feel like we should be confident about what we're putting into a guide. After all, the reader is trusting that the information we've taken time to accumulate is our best-in-class "seal of approval" technique.

We can deliberate as much as we want while writing it, and leave only the information we know is tried and true in the document. There's no reason to clue the reader in on this process, the work should already be done.

Reply:
Valid point, for now I have made this
```
# One of the easiest ways to do this, ...
```

I will however leave this conversation open and ask for further feedback from other docs contributors.